incubation_process.md

```diff
@@ -41,20 +42,22 @@ The barrier for entering the Sandbox stage is intended to be low, so that it's e
 
 ## Stage 2: Incubation
```

Comment:
General comment: Do we have to precise that the project candidate has to be at the previous stage to reach the next ?

Reply:
For me, this is implicitly evident from the description, but we can formulate it explicitly. A 'skipping' of stages doesn't make sense since the first step requires a formal confirmation from the Board of Directors and the stages build upon each other.

Reply:
The diagram with the stages shows that they are sequential, but it probably is a good idea to clarify this in the text as well. I will add a sentence.